dev/airflow-github

```diff
@@ -46,7 +46,7 @@ except ImportError:
     sys.exit(-1)
 
 STATUS_COLOR_MAP = {
-    'Resolved': 'green',
+    'Closed': 'green',
     'Open': 'red',
```

Comment:
I wonder if it’s a good idea to change the colors because this is now entirely opposite to GitHub’s scheme

Reply:
I think the idea is to color it in the way that's meaningful for the release manager, not to match GitHub's colors.

Red -> action needed
Green -> no action needed.